### ISSUE TITLE
fix linting errors

### ISSUE THREAD

**Issue by kfarr** [3DStreet/3dstreet#527]
there are many linting errors that require manual work to fix them

**Comment by Algorush** [comment]
Now eslint is configured more strictly. Therefore, I will list some controversial points that had to be made.
- About different versions of `json-utils`. I think we can leave only last one now? I'll delete the rest.
- In `index.js` I'll comment:
```
if (typeof VERSION !== 'undefined') {
  console.log(`3DStreet Version: ${VERSION}`);
}
```

**Comment by vincentfretin** [comment]
Yes keep only json-utils_1.1.js that's the one used, you can remove the other versions.

Keep the 3 lines, just add `/* global VERSION */` on top. webpack is setting this VERSION variable.

**Comment by Algorush** [comment]
> Yes keep only json-utils_1.1.js that's the one used, you can remove the other versions.
> 
> Keep the 3 lines, just add `/* global VERSION */` on top. webpack is setting this VERSION variable.

Ok, thanks. I have problems running the command in my localhost: `npm run lint`.
```
No files matching the pattern "'src/**/*.js'" were found.
Please check for typing mistakes in the pattern.
```
Although I run it from 3DStreet root dir.

**Comment by Algorush** [comment]
> That's weird. Which OS are you using? Windows? Can you try to change the lint command to `eslint --ext .js,.jsx src` to see if that helps?

Yes, windows. The problem went away when I removed the quotes here: `'src/**/*.js'`
Or if escape double quotes, it will work on unix/windows, as chatGPT says:
`"lint": "eslint \"src/**/*.js\" \"src/**/*.jsx\"",`

**Comment by vincentfretin** [comment]
Ok, let's change the command to `eslint --ext .js,.jsx src` there is no quotes in it.

**Comment by Algorush** [comment]
Its is done now: https://github.com/3DStreet/3dstreet/pull/530

**Comment by kfarr** [comment]
closed by #530
